app/stores/toplevel/ProfileStore.js

```diff
@@ -93,6 +128,32 @@ export default class SettingsStore extends Store {
     await this.getProfileLocaleRequest.execute(); // eagerly cache
   };
 
+  _updateTheme = async ({ theme }: { theme: string }) => {
+    await this.unsetCustomThemeRequest.execute();
```

Comment:
I'm confused why you play around with custom theme here. I think both of these lines should be removed.

Reply:
yep already committed it earlier today